CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Welcome, contributors to DQ Robotics!
+
+We are happy about your interest in our project! DQ Robotics is a standalone, **fully moderated** open-source library, and contributions are always welcome! 
+
+If your proposed modifications change current functionality, you **must** first discuss them with the developers team and ensure they are approved. Otherwise, it will be rejected without review.
+
+This is a set of guidelines for contributing to [DQ Robotics](https://dqrobotics.github.io/).
+
+# Workflow
+
+- Fork the [master branch of dqrobotics/matlab](https://github.com/dqrobotics/matlab).
+- Propose your modifications and open a draft pull request. Keep in mind the following **requirements**:
+   - Propose individual changes (several changes of the same type are allowed on the same pull request).
+   - Do not unnecessarily change any internal implementation that is working correctly without prior approval.
+   - Include a clear and concise rationale behind each pull request.
+- Your modifications will be tested automatically through Github actions. Github actions run the tests of [matlab-tests](https://github.com/dqrobotics/matlab-tests), which execute all the examples of [matlab-examples](https://github.com/dqrobotics/matlab-examples). 
+- Once your draft pull request passes all the tests, you can switch the status to pull request. (More details [here](https://github.blog/2019-02-14-introducing-draft-pull-requests/)).
+
+![Untitled Diagram](https://user-images.githubusercontent.com/23158313/150255548-b4475747-444e-4530-9b20-def1655bb4f9.png)
```

Comment:
Very good diagram, @juanjqo. Please, make the following amendments:

- Does pass all tests? -> Does it pass all tests?
- It is your PR incompatible with the following repositories?  -> Is your PR incompatible with the following repositories?
- Does pass all tests in the master branch? -> Does it pass all the tests in the master branch?

Reply:
- [ ] Done.

Reply:
@juanjqo, the link above [(https://user-images.githubusercontent.com/23158313/150255548-b4475747-444e-4530-9b20-def1655bb4f9.png)](https://user-images.githubusercontent.com/23158313/150255548-b4475747-444e-4530-9b20-def1655bb4f9.png) still points to the outdated image. Please make sure you double-check your changes to avoid others having unnecessary double-work.

Reply:
Hi @bvadorno, 

I do not know why Github is not showing the whole history. I implemented the modifications in [be5d975](https://github.com/juanjqo/matlab/commit/be5d9755fcdfea3511b1efbb0f2da89e06ee3ccc) six days ago. In that commit I replaced the link
`![Untitled Diagram](https://user-images.githubusercontent.com/23158313/150255548-b4475747-444e-4530-9b20-def1655bb4f9.png)`
by
`<img src="https://user-images.githubusercontent.com/23158313/205527140-74e7ad26-17b3-4f2c-a159-fca7c52a42ea.png" alt="drawing" width="1200"/>`.

However, I realized a small bug here (the letter `p` in `passing` was hidden).

<img width="1135" alt="Screenshot 2022-12-11 at 15 50 51" src="https://user-images.githubusercontent.com/23158313/206890419-af390f87-5fd0-4273-a950-33f41ba5d056.png">

I'm going to fixed it.